scripts/assets/options.go.templ

```diff
@@ -115,11 +117,12 @@ func UpdateFieldsForJSON(flags *pflag.FlagSet) (*serverSettingsForJSON, error) {
 func GetCommandLineArgsForStartCommand(flags *pflag.FlagSet) ([]string, error) {
 	var commandLineArgs []string
   <%_ for (const { propertyName, capitalizedName, aliasFor, enums, valuePart } of commandFlags) {
+      const kebabPropertyName = kebabConvert(propertyName);
       const actualPropertyName = aliasFor || propertyName;
```

Comment:
It would be good to have a comment noting that it's _not_ kebab-converted, because the app takes them that way. (Actually, around line 129 where it gets used.)

Reply:
done